packages/search-field/src/SearchField.js

```diff
@@ -224,6 +235,7 @@ class SearchField extends React.Component {
           onChange={this.onSearchChange}
           type={selectedItem ? "text" : "search"}
           clearable
+          focused={autoFocus}
```

Comment:
Is this covered by storyshots? If not, we must add an unit test here...

Reply:
This boolean `autoFocus` is covered in the `with select item` in Storybook first as `undefined`, and if we clear the field, `autoFocus` returns true, then false. Is this enough?